pkg/image/driver.go

```diff
@@ -66,19 +64,6 @@ type Driver interface {
 // drivers holds all registered image drivers
 var drivers = make(map[string]Driver)
 
-// RegisterDriver registers an image driver by name.
-func RegisterDriver(name string, driver Driver) error {
```

Comment:
This is important for this package API to allow plugins to register different types of image drivers: https://github.com/apptainer/apptainer/blob/main/examples/plugins/ubuntu-userns-overlay-plugin/main.go#L42

Reply:
will preserve -- interesting that the dependency walker I use did not detect this.